script/api.py

```diff
@@ -103,12 +125,87 @@ def login():
 
     if session.get("user_id") is None:
         session["user_id"] = user_id
-        return jsonify({"Username": username})
+        return jsonify({"username": username, "token": "test123"})
 
     if not session["user_id"] == user_id:
         return jsonify({"message": "Bad Request"}), 400
 
-    return jsonify({"username": username})
+    return jsonify({"username": username, "token": "test123"})
+
+
+@app.route("/forgot_password", methods=["POST"])
+def forgot_password():
+    """FORGOT PASSWORD API
+    Parameters:
+    email (String): Email of user
+
+    Returns:
+        Http Response: {
+            String : It can be either Message or username
+            Integer : Status Code
+        }
+    """
+    email = request.json.get("email")
+    cur.execute("SELECT * FROM users WHERE email='{}'".format(email))
+    user = cur.fetchone()
+    if not user:
+        return jsonify({"message": "User doesn't exist."})
+    token = s.dumps(email)
+
+    link = url_for(
+        "confirm_email",
+        token=token,
+        _external=False,
+    )
+    msg = Message(
+        "GeoVisualier Password Reset",
+        sender="shalinisharma1297@gmail.com",
+        recipients=[email],
+    )
+    msg.body = "Hello,\nPlease reset your password using this: http://localhost:3000{}\nThanks".format(
+        link
+    )
+
+    mail.send(msg)
+
+    return jsonify({"message": "Email sent successfully."})
+
+
+@app.route("/confirm_email/<token>", methods=["POST"])
+def confirm_email(token):
+    """RESET PASSWORD API
+    Parameters:
+    token (String): token
+    password (String): Password of user
+    password (String): Password of user
+
+    Returns:
+        Http Response: {
+            String : It can be either Message or username
+            Integer : Status Code
+        }
+    """
+    password = request.json.get("password")
+    passwordAgain = request.json.get("passwordAgain")
```

Comment:
```suggestion
    password_again = request.json.get("passwordAgain")
```
This can be named better as confirm_password